### ISSUE TITLE
[BUG]: Error running on versions of Python < 3.8

### ISSUE THREAD

**Issue by bsharper** [wasi-master/sponsorblock.py#4]
### Description of the bug

When attempting to import this library on Python lower than 3.8, the following error will appear:

`ImportError: cannot import name 'Literal' from 'typing' (/usr/lib/python3.7/typing.py)`

This error was reproduced on 2 of my Raspberry Pis running Raspbian GNU/Linux 10 (buster).
There is fortunately a trivial fix. In `utils.py` change

```
from typing import Dict, Literal
```

to 

```
from typing import Dict
from typing_extensions import Literal
```
Or put the current `Literal` import code in a try / except with the exception importing the 2nd way. I could make a pull request but figured this would be easier.

### Steps To Reproduce

On Python < 3.8:

```
pip install sponsorblock
python -c "import sponsorblock"
```

**Comment by wasi-master** [comment]
Thanks for the issue. It's 2 AM here for me and I'm in my bed trying to sleep. I'll have to fix this in about 2-3 hours when I get up to eat my Sehri. Sorry for the delay but I really need to sleep and can't use my PC now

**Comment by bsharper** [comment]
Ramadan Mubarak! I appreciate the quick reply. Sleep is much more important than fixing this bug that most people probably won't encounter. 

Thank you for this project, overall it's been working beautifully.

**Comment by wasi-master** [comment]
Fixed in version `0.2.2`

**Comment by bsharper** [comment]
Confirmed fixed. Thanks!
